src/app/document.js

```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { AfterRoot, AfterData } from '@jaredpalmer/after';
+
+class Document extends React.Component {
+  static propTypes = {
```

Comment:
The `object` prop type is banned, will fail linting. Need to use `shape` or `objectOf`, see: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/forbid-prop-types.md

Also, I think all this prop types are required? If so, append `.isRequired`

Reply:
👍